bundles/binding/org.openhab.binding.zwave/database/widom/wps104.xml

```diff
@@ -0,0 +1,636 @@
+			Available Range: 1 min - 100 mins</Help>
+		</Parameter>
+		<Parameter>
+			<Index>47</Index>
+			<Type>list</Type>
+			<Default>3</Default>
+			<Size>1</Size>
+			<Label>Variation Total Energy</Label>
+			<Help lang="en">Defines the increase of total energy required to generate a report listing the level 
+			of total energy to the devices associated to the Energy Notification Group.</Help>
+			<Item>	
+				<Value>0</Value>
+				<Label lang="en">IGNORE: No report provided.</Label>
+			</Item>
+			<Item>	
+				<Value>1</Value>
+				<Label lang="en">Report at 1 thousandth of KWh</Label>
+			</Item>
+			<Item>	
+				<Value>2</Value>
+				<Label lang="en">Report at 1 hundredth of KWh.</Label>
+			</Item>
+			<Item>	
+				<Value>3</Value>
+				<Label lang="en">Report at 1 tenth of KWh</Label>
+			</Item>
+			<Item>	
+				<Value>4</Value>
+				<Label lang="en">Report at 1 KWh</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>48</Index>
+			<Type>byte</Type>
+			<Default>2300</Default>
+			<Minimum>1100</Minimum>
+			<Maximum>2500</Maximum>
+			<Size>2</Size>
+			<Label lang="en">Nominal Voltage</Label>
+			<Help lang="en">Defines the nominal voltage value in tenths of volts. Together with parameter No. 49 
+			this is used by the automatic notification system to send reports on variations of voltage.
+			Available Range: 1100 v - 2500 v</Help>
+		</Parameter>
+		<Parameter>
+			<Index>49</Index>
+			<Type>byte</Type>
+			<Default>10</Default>
+			<Minimum>1</Minimum>
+			<Maximum>100</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Fall In Maximum Voltage</Label>
+			<Help lang="en">Defines, as a percentage, the nominal value for permitted fall in maximum voltage.
+			Available Range: 1 - 100%</Help>
+		</Parameter>
+		<Parameter>
+			<Index>50</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Minimum>0</Minimum>
+			<Maximum>62</Maximum>
+			<Size>2</Size>
+			<Label lang="en">Electric parameters subjected to automatic notification</Label>
+			<Help lang="en">Defines which electric parameters, other than power, will be subjected to automatic 
+			notification. The value to be set for this parameter must be calculated as the sum of values 
+			associated to the individual electric parameter: Power=0, Energy=2, Voltage=4, Current=8, 
+			Power factor=16, Multilevel Sensor: Power Report=32 To send report only for Power please set 
+			the value to 0. By default the report are sent for (Power, Energy, Voltage, Current, Power Factor)
+			 0+2+4+8+16=30</Help>
+		</Parameter>
+		<Parameter>
+			<Index>51</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Size>2</Size>
+			<Label lang="en">Voltage RMS value</Label>
+			<Help lang="en">Reading of this parameter returns the value of voltage RMS at the time of the reading 
+			expressed in tenths of Volts.</Help>
+			<ReadOnly>true</ReadOnly>
+		</Parameter>
+		<Parameter>
+			<Index>52</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Size>2</Size>
```

Comment:
This should have the min/max elements. If the manual doesn't state them, then just use the min/max of the size (ie 0 and 65535 for 2 bytes)